docs/csharp/quick-starts/branches-and-loops-local.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Quick Start - Branches and lops - C# Guide
+description: In this quick start about branches and loops, you'll write C# code to explore the language syntax that supports conditional branches and loops to repeat statements.
+author: billwagner
+ms.author: wiwagn
+ms.date: 10/31/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+ms.custom: mvc
+---
+
+# Branches and loops
+
+This quick start teaches you about how to write code that examines variables and changes execution path based on those variables. You'll write C# and see the results of compiling and running your code. It contains a series of lessons that explore branching and looping constructs in C#. These lessons teach you the fundamentals of the C# language.
+
+This quick start expects you to have a machine you can use for development. The .NET topic [Get Started in 10 minutes](https://www.microsoft.com/net/core) has instructions for setting up your local development environment on Mac, PC or Linux.
+
+## Make decisions using the `if` statement
+
+Create a directory named **branches-quickstart**. Make that the current directory and run `dotnet new console -n BranchesAndLoops -o .`.
+
+Open **Program.cs** in your favorite editor, and replace the line `Console.Writeline("Hello World!");` with the following:
+
+```csharp
+int a = 5;
+int b = 6;
+if (a + b > 10)
+    Console.WriteLine("The answer is greater than 10.");
+```
+
+Try this code by typing `dotnet run` in the your console window.
+
+Modify the declaration of `b` so that the sum is less than 10: 
+
+```csharp
+int b = 3;
+```
+
+Type `dotnet run` again. Because the answer is less than 10, nothing is printed. The **condition** you're testing is false. You don't have any code to execute because you've only
+written one of the possible branches for an `if` statement: the true branch.
+
+> [!TIP]
+> As you explore C# (or any programming language), you'll
```

Comment:
After responding more than a hundred times to  livefyre with:
Wow, awesome error message!  - I'd have to write the preceding as:

 As you explore programming language, you'll
  make mistakes coding mistake. The **compiler** will
  find and report the errors. Look closely the the error output and the code that generated the error. The compler error can usually help you find the problem.